dev-libs/zthread/files/zthread-2.3.2-clang.patch

```diff
@@ -0,0 +1,12 @@
+diff -r 2a43e83684d5 src/foreign/zthread/include/zthread/Guard.h
```

Comment:
Also, if possible, could you give a short description in the header of the patch?
https://dev.gentoo.org/~vapier/clean-patches